tensorflow-keras/src/main/java/org/tensorflow/keras/backend/tf/ControlDependencies.java

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.keras.backend.tf;
+
+import org.tensorflow.keras.backend.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import org.tensorflow.Operand;
+import org.tensorflow.op.Op;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.family.TType;
+
+/** Container for ControlDepencies, so that the primary Operand is remembered. */
+public class ControlDependencies {
```

Comment:
I don't think we need this class, as the user can very easily add its control dependencies using directly `tf.withControlDependencies(...)`, unless you have another use case in mind?

Reply:
org.tensorflow.keras.backend.K is a part of Keras, the sub packages the I used for under backend were parking places for classes that probably be in other tensorflow modules.

Reply:
ControlDependencies is just a utility class to create control dependencies. It was manly put in to avoid rewriting all that code all the time.

Reply:
If `ControlDependencies` is a utility for the keras package we could make it package private? We can always make it public later if we want to expose the functionality.

Reply:
I removed ControlDependencies from this PR. I may revisit it when new PRs are created for other Keras features.